pkgs/development/tools/analysis/rizin/default.nix

```
@@ -26,11 +25,11 @@
 
 let rizin = stdenv.mkDerivation rec {
   pname = "rizin";
-  version = "0.6.3";
+  version = "0.7.1";
 
   src = fetchurl {
     url = "https://github.com/rizinorg/rizin/releases/download/v${version}/rizin-src-v${version}.tar.xz";
-    hash = "sha256-lfZMarnm2qnp+lY0OY649s206/LoFNouTLlp0x9FCcI=";
+    hash = "sha256-FJ3I7tQHAIm25OZQcdVfVxwNLkxy0u5CBWKiMhMIwpQ=";
   };
 
   mesonFlags = [
```

```suggestion
  mesonFlags = [
    "-Duse_sys_libzstd=enabled"
    "-Duse_sys_pcre2=enabled"
```

(except please put `use_sys_libzstd` after `use_sys_lz4` and `use_sys_pcre2` after `use_sys_tree_sitter`, the way it's ordered in meson_options.txt, I can't do it with Github suggestions)